src/main/bash/sdkman-selfupdate.sh

```diff
@@ -48,7 +48,7 @@ function __sdkman_auto_update() {
 		__sdkman_echo_no_colour "The current version is $remote_version, but you have $version."
 		echo ""
 
-		if [[ "$sdkman_auto_selfupdate" != "true" ]]; then
+		if [[ "$sdkman_auto_answer" != "true" ]]; then
```

Comment:
Double negative again. How about `$sdkman_auto_answer == false`?

Reply:
Same as above. To prevent issues when the user do not have such entry in its config file.